primo/data_parser/well_data.py

```diff
@@ -479,6 +483,36 @@ def flag_wells(self, rows: list, col_name: str):
         self.data[col_name] = 0
         self.data.loc[rows, col_name] = 1
 
+    def add_new_column_ordered(
+        self, col_name_list: list, values: Union[np.array, pd.DataFrame, list]
+    ):
+        """
+        Adds a single column to the WellData and other related data structures
+        The column must be in the correct order corresponding to the existing data
+
+        Parameters
+        ----------
+        col_name : list(str)
+            List of 2 strings: the column variable name, the column header for the data
+        values : np.array, pd.DataFrame, list
+            The values for the column
+        """
+        if len(col_name_list) != 2:
```

Comment:
Minor, but curious why this must be a list? IF we are forcing this list to be of 2 strings, why not have them separately as 2 string inputs?

Reply:
This is a good point. Incidentally, after removing the distance to centroid computation, this function isn't used, but it could possibly be a useful function for the future. Should we keep it or remove it?

Reply:
^^^ jk. found one spot it is used. Keeping it.